TestFoundation/TestTimeZone.swift

```diff
@@ -166,13 +166,15 @@ class TestTimeZone: XCTestCase {
     }
 
     func test_systemTimeZoneUsesSystemTime() {
+#if !os(Windows)
```

Comment:
Let’s also bracket this test’s entry in `allTests` with `!os(Windows)`, so that people aren’t misled into thinking this test works there.